### DISCUSSION TITLE
Prism customization

### DISCUSSION THREAD

**Discussion by Polatouche** [compodoc/compodoc#1048] [Ideas]
Hi,

first, thank you for this nice tool.
To add syntax hilighting for non default langage as apacheconf, sql, php, ... I replace the prism.js and prism.css by a customized version downloaded from [prismjs.com](https://prismjs.com/download.html).
Can it be done automatiquely by compodoc, using an option like `-prism customprismdirectory `?
This will be useful for writing additional documentation using markdown format.

Thank you!

**Comment by vogloblinsky** [comment]
Why did you have to change that ?

**Reply by Polatouche** [reply]
Hi,
because I'm using compodoc to document a web app. The front end is made with angular, the back-end with php/msql.
I use compodoc to document also the back-end and the server configuration using additional markdown files displaying php code, sql code and .htaccess files.
I think other web apps may use other langages as well for the back end and can by this way document the back end using compodoc.

Thank you!

**Comment by vogloblinsky** [comment]
More modern languages support added in 1.1.12 coming this week.
